tests/queries/0_stateless/01622_defaults_for_file_engine.sql

```diff
@@ -0,0 +1,7 @@
+insert into table function file("data1622.json", "TSV", "value String") VALUES ('{"a":1}');
+drop table if exists json;
+create table json(a int, b int default 7) engine File(JSONEachRow, 'data1622.json');
```

Comment:
Maybe also add `c default a + b`? Just to be sure :)

Reply:
Added